pkg/flags/environment.go

```diff
@@ -0,0 +1,84 @@
+package flags
+
+import "github.com/spf13/pflag"
+
+type EnvironmentalFlags struct {
+	Platform     string
+	Network      string
+	Upgrade      string
+	Topology     string
+	Architecture string
+	Installer    string
+	Config       []string
```

Comment:
Could we drop `Config` for now? We're not going to do anything with them for a while